scribble-devs/excalidraw-room
# Add server-side Mixpanel analytics

Add tracking in Mixpanel for the following metrics:
1. Maximum room size
2. Duration of collaboration (amount of time room contains > 1 user)
3. Duration of room (amount of time room contains >= 1 user)

Tested locally:
```
1. Run app locally with: docker-compose up --build
2. Start session, start timer
3. Open up link in incognito window, note time
4. Open up link in Firefox
5. Close Firefox
6. Close incognito, note time
7. Reopen incognito, note time
8. Close incognito, note time
9. Close room, note time
10. Monitor logs for errors
11. Monitor Mixpanel for expected events

RESULTS
3. 15s
6. 50s
7. 1:10
8. 1:50
9. 2:00
10. No errors
11. 
Room collaboration ended
max_size: 3
duration_seconds: 34s

Room collaboration ended
max_size: 2
duration_seconds: 38s

Room closed
max_size: 3
duration_seconds: 120s
```
